### PR TITLE
Feature/mobile iron doc

### PR DESCRIPTION
**Pull request by julsemaan**
For review + merge

### CODE REVIEW

File: docs/PacketFence_MobileIron_Quick_Install_Guide.asciidoc [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+================================================
+:encoding: UTF-8
+:lang: en
+:doctype: book
+
+include::includes/global-attributes.asciidoc[]
+
+About this Guide
+----------------
+This guide has been created in order to help sales engineers, product managers, or network specialists demonstrate the PacketFence capabilities on-site with an existing or potential customer.  It can also provide guidelines to setup a proof of concept for a potential PacketFence deployment using the Mobile Iron mobile device manager.
```
Comment (extrafu) [inline docs/PacketFence_MobileIron_Quick_Install_Guide.asciidoc]: Mobile Iron -> MobileIron (everywhere)

File: docs/PacketFence_MobileIron_Quick_Install_Guide.asciidoc [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+MDM profile
+^^^^^^^^^^^
+One important step is to enable the MDM profile like in this screenshot. 
+Note that this will require you to create an MDM certificate with Apple. Refer to the Mobile Iron documentation for specifics about this step.
+
+image::docs/images/mobile-iron-mdm-profile-config.png[scaledwidth="100%",alt="MDM profile configuration"]
+
+Step 2: Create an API user
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+Next, we'll need a user that has the rights to access the Mobile Iron API in order to verify the state of the devices directly from PacketFence.
```
Comment (extrafu) [inline docs/PacketFence_MobileIron_Quick_Install_Guide.asciidoc]: we'll -> we will

File: docs/PacketFence_MobileIron_Quick_Install_Guide.asciidoc [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+Now go in the ADMIN tab, check the box next to your newly created user and then in Actions select Assign to space
+
+image::docs/images/mobile-iron-go-to-assign-space.png[scaledwidth="100%",alt="Go to assign space"]
+
+Select the Global space at the top and then check API at the bottom.
+You should now see API in the roles list of your newly created user when viewing the users list.
+
+Step 3: Gather the boarding host
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To find the boarding host, add a fake device to Mobile Iron and at the end of the process you'll see the registration instructions.
```
Comment (extrafu) [inline docs/PacketFence_MobileIron_Quick_Install_Guide.asciidoc]: you'll -> you will

File: docs/PacketFence_MobileIron_Quick_Install_Guide.asciidoc [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+Here are the URIs that should work by default. 
+Replace accountName by your real account/instance name at Mobile Iron.
+
+* Android : https://m.mobileiron.net/accountName/c/d/android.html
+* IOS devices : https://m.mobileiron.net/accountName/c/d/ios.html
+* Windows : https://m.mobileiron.net/accountName/EnrollmentServer/Discovery.svc
+
+Add the provisioner to the portal profile
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+In order for the provisionner to be used by your captive portal you need to add it in it's configuration.
```
Comment (extrafu) [inline docs/PacketFence_MobileIron_Quick_Install_Guide.asciidoc]: it's -> its

File: docs/PacketFence_MobileIron_Quick_Install_Guide.asciidoc [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+* You have access to a Mobile Iron cloud account
+
+Quick installation
+------------------
+Step 1: Configure Mobile Iron
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+First of all you will need to configure the basic functionality of Mobile Iron using their documentation.
+
+MDM profile
+^^^^^^^^^^^
```
Comment (cgx) [inline docs/PacketFence_MobileIron_Quick_Install_Guide.asciidoc]: I guess this title should be one level down (`+++`).

File: docs/PacketFence_MobileIron_Quick_Install_Guide.asciidoc [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+
+image::docs/images/mobile-iron-boarding.png[scaledwidth="100%",alt="Boarding info"]
+
+Step 3: Configure PacketFence
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In PacketFence, MDM are refered to as provisioners. 
+This will walk you through adding MobileIron as a provisioner.
+
+Create the provisioner
+^^^^^^^^^^^^^^^^^^^^^^
```
Comment (cgx) [inline docs/PacketFence_MobileIron_Quick_Install_Guide.asciidoc]: I guess this title should be one level down (`+++`).

File: docs/PacketFence_MobileIron_Quick_Install_Guide.asciidoc [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+
+Here are the URIs that should work by default. 
+Replace accountName by your real account/instance name at Mobile Iron.
+
+* Android : https://m.mobileiron.net/accountName/c/d/android.html
+* IOS devices : https://m.mobileiron.net/accountName/c/d/ios.html
+* Windows : https://m.mobileiron.net/accountName/EnrollmentServer/Discovery.svc
+
+Add the provisioner to the portal profile
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (cgx) [inline docs/PacketFence_MobileIron_Quick_Install_Guide.asciidoc]: I guess this title should be one level down (`+++`).

**Pull request merged**